db/migrate/20230228014101_add_first_approved_at_to_review.rb

```diff
@@ -0,0 +1,8 @@
+# typed: true
+# frozen_string_literal: true
+
+class AddFirstApprovedAtToReview < ActiveRecord::Migration[7.0]
+  def change
+    add_column(:reviews, :first_approved_at, :datetime, default: nil)
```

Comment:
For migration, we should add first_approved to all existing approved reviews. Is it possible to add a second migration that does this?